src/constants.js

```diff
@@ -0,0 +1,230 @@
+const offerTypes = {
+  TAXI: 'Taxi',
+  BUS:'Bus',
+  TRAIN:'Train',
+  SHIP:'Ship',
+  DRIVE:'Drive',
+  FLIGHT:'Flight',
+  CHECKIN:'Check-in',
+  SIGHTSEENING:'Sightseeing',
+  RESTARAUNT:'Restaurant'
+};
+
+const OFFERS_TYPES = Object.values(offerTypes);
+
+const CITY = [
+  'Geneva',
+  'Chamonix',
+  'Amsterdam',
+  'Dublin',
+  'Lisbon',
+  'Berlin',
+];
+
+const EVENT_POINT_COUNT = 15;
+
+const additionalOffers = [
+  {
+    'type': 'Taxi',
+    'offers': [
+      {
+        'id': 1,
+        'title': 'Upgrade to a business class',
+        'price': 120
+      },
+      {
+        'id': 2,
+        'title': 'Choose the radio station',
+        'price': 60
+      },
+      {
+        'id': 3,
+        'title': 'Order Uber',
+        'price': 20
+      },
+    ]
+  },
+  {
+    'type': 'Bus',
+    'offers': [
+      {
+        'id': 1,
+        'title': 'Upgrade to a business class',
+        'price': 120
+      },
+      {
+        'id': 2,
+        'title': 'Rent mini-bus',
+        'price': 500
+      }
+    ]
+  },
+  {
+    'type': 'Train',
+    'offers': [
+      {
+        'id': 1,
+        'title': 'Upgrade to a business class',
+        'price': 120
+      },
+      {
+        'id': 2,
+        'title': 'Order breakfast',
+        'price': 60
+      }
+    ]
+  },
+  {
+    'type': 'Ship',
+    'offers': [
+      {
+        'id': 1,
+        'title': 'Upgrade to a business class',
+        'price': 120
+      },
+      {
+        'id': 2,
+        'title': 'Order breakfast',
+        'price': 60
+      }
+    ]
+  },
+  {
+    'type': 'Drive',
+    'offers': [
+      {
+        'id': 1,
+        'title': 'Детское кресло',
+        'price': 200
+      },
+      {
+        'id': 2,
+        'title': 'Курить в салоне',
+        'price': 500
+      }
+    ]
+  },
+  {
+    'type': 'Flight',
+    'offers': [
+      {
+        'id': 1,
+        'title': 'Негабаритный багаж',
+        'price': 60
+      },
+      {
+        'id': 2,
+        'title': 'Ранее бронирование',
+        'price': 60
+      }
+    ]
+  },
+  {
+    'type': 'Check-in',
+    'offers': [
+      {
+        'id': 1,
+        'title': 'Check-in online',
+        'price': 20
+      },
+      {
+        'id': 2,
+        'title': 'Поздний выезд',
+        'price': 50
+      },
+    ]
+  },
+  {
+    'type': 'Sightseeing',
+    'offers': [
+      {
+        'id': 1,
+        'title': 'Аудио гид',
+        'price': 50
+      },
+      {
+        'id': 2,
+        'title': 'Персональный гид',
+        'price': 200
+      }
+    ]
+  },
+  {
+    'type': 'Restaurant',
+    'offers': [
+      {
+        'id': 1,
+        'title': 'Забронировать столик',
+        'price': 5
+      },
+      {
+        'id': 2,
+        'title': 'Специальное меню',
+        'price': 10
+      }
+    ]
+  },
+];
+
+const POINT_PRICE = {
+  'Taxi': {
```

Comment:
```suggestion
  [OfferType.TAXI]: {
```
И даже тут можно. Видишь, объявляем в одном месте - переиспользуем везде везде